src/fragments/lib-v1/auth/native_common/user_attributes/common.mdx

```diff
@@ -46,9 +46,9 @@ import flutter8 from '/src/fragments/lib-v1/auth/flutter/user_attributes/30_conf
 
 <Fragments fragments={{ flutter: flutter8 }} />
 
-## Resend verification code
+## Send user attribute verification code
 
-If the code has expired or the user needs to resend the confirmation code, invoke the resend api as shown below:
+If an attribute needs to be verified while the user is authenticated, invoke the send api as shown below:
```

Comment:
```suggestion
If an attribute needs to be verified while the user is authenticated, invoke the api as shown below:
```

Let's remove the reference to the API name here. The API is still called `resendUserAttributeConfirmationCode` on most platforms in vPrev but is called `sendUserAttributeVerificationCode` for flutter.